File: python/1045.py
# -*- coding: utf-8 -*-

n = [float(i) for i in input().split()]
n.sort(reverse=True)
A = n[0]
B = n[1]
C = n[2]

if A>=B+C:
    print('NAO FORMA TRIANGULO')
else:
    if (A**2)==(B**2)+(C**2):
        print('TRIANGULO RETANGULO')
    if (A**2)>(B**2)+(C**2):
        print('TRIANGULO OBTUSANGULO')
    if (A**2)<(B**2)+(C**2):
        print('TRIANGULO ACUTANGULO')
    if A==B==C:
        print('TRIANGULO EQUILATERO')
    if A==B!=C or B==C!=A or A==C!=B:
        print('TRIANGULO ISOSCELES')

File: categorias/iniciante/c++/1045.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    double a, b, c, tmp;

    cin >> a >> b >> c;

    if (a < b) {
        tmp = a;
        a = b;
        b = tmp;
    }
    if (b < c) {
        tmp = c;
        c = b;
        b = tmp;
    }
    if (a < b) {
        tmp = a;
        a = b;
        b = tmp;
    }

    if (a >= b+c) {
        cout << "NAO FORMA TRIANGULO\n";
    } else {
        if (pow(a, 2) == pow(b, 2) + pow(c, 2)) {
            cout << "TRIANGULO RETANGULO\n";
        }
        if (pow(a, 2) > pow(b, 2) + pow(c, 2)) {
            cout << "TRIANGULO OBTUSANGULO\n";
        }
        if (pow(a, 2) < pow(b, 2) + pow(c, 2)) {
            cout << "TRIANGULO ACUTANGULO\n";
        }
        if (a==b && b==c) {
            cout << "TRIANGULO EQUILATERO\n";
        } else if (
            a==b ||
            b==c ||
            a==c
        ) {
            cout << "TRIANGULO ISOSCELES\n";
        }
    }

    return 0;
}